templates/services/gateway.tmpl.yaml

```diff
@@ -52,6 +52,11 @@ data:
         proxy_set_header x-ats-namespace $deviceNamespace;
         proxy_pass http://device-registry;
       }
+      location /system_info/network {
```

Comment:
We can rewrite all `/system_info/` endpoints to device-registry. Aktualizr can access these instead of the  legacy `/core/system_info`.

Reply:
I think that would be great. I'd tried to get some traction on that method with aktualizr:

 https://github.com/advancedtelematic/aktualizr/pull/716

but haven't heard anything. Shall we table this and wait on a change in that project first?